step-issuer-compat.yaml

```
@@ -0,0 +1,20 @@
+package:
+  name: step-issuer-compat
+  version: 0.0.1
+  epoch: 0
+  description: step-issuer compatability
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+
+pipeline:
+  - runs: |
+      cd ${{targets.destdir}}
+      ln -s /usr/bin/step-issuer manager
```

```suggestion
pipeline:
  - runs: |
      ln -sf /usr/bin/step-issuer ${{targets.destdir}}/manager
```

Try this and see if that helps. Ideally this will be a subpackage in `step-issuer`, with a requires to pull in the main package, but for testing this ought to work.